benchmarks/project/pom.xml

```diff
@@ -128,6 +129,17 @@
         <artifactId>fmt-maven-plugin</artifactId>
       </plugin>
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
```

Comment:
This plugin doesn't work Java 17? Is this something that was already on our radar? I'm not sure we'll want to keep it disabled, no?

Reply:
It was already on my radar. Maybe it was badly worded by me, what doesn't work is `maven-dependency-analyzer`. We already talked about this and came to the conclusion that it's okay to disable this for the modules that we upgrade to Java 17 und re-enable when a fixed version is released.